src/Entity/GeneratedBook.php

```diff
@@ -128,4 +139,18 @@ public function getFormat(): string {
 	public function setFormat( string $format ): void {
 		$this->format = $format;
 	}
+
+	/**
+	 * Get the generation duration in seconds.
+	 * @return int|null
+	 */
+	public function getDuration(): ?int {
+		return $this->duration;
+	}
+
+	public function setDuration( ?int $duration ): self {
```

Comment:
I'm guessing you used the Symfony CLI to create these methods when you added the column. Weird how it had it return `self` when it didn't for me! Not a big deal, but I would just make it `void` to be consistent with the other methods.

Also missing the doc block.

Reply:
Oh yeah, that is strange. Fixed both things.